README_dev.md

```diff
@@ -21,9 +21,9 @@ This workspace provide two types of scripts:
 ## Linting
 
 -   Typescript
-    -   each package provides its own typescript linting script. You can run it by `npm run lint:ts` from the package directory or by `npm run -w packages/<package> lint` from the root of the project
-    -   the root project also provides a script that lints all packages. You can run it by `npm run lint:ts` from the root of the project
--   Prettier: simply run `npm run prettier` from the root of the project
+    -   each package provides its own typescript linting script. You can run it by `npm run lint:tsc` from the package directory or by `npm run -w packages/<package> lint` from the root of the project
```

Comment:
The second occurrence here misses the `:tsc` -> `npm run -w packages/<package> lint:tsc`